docs/docs/module_guides/models/embeddings.md

```diff
@@ -24,10 +24,12 @@ Then:
 
 ```python
 from llama_index.embeddings.openai import OpenAIEmbedding
+from llama_index.core import VectorStoreIndex
 from llama_index.core import Settings
 
+embed_model = OpenAIEmbedding()
 # global
-Settings.embed_model = OpenAIEmbedding()
+Settings.embed_model = embed_model
```

Comment:
this wasn't an error tho right? Just the missing import for `VectorStoreIndex`?

Reply:
i reverted this back to the original!

Reply:
Yeah this wasn't.. just tried to reuse variable